src/components/BigButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+interface BigButtonProps {
+  color?: string;
```

Comment:
I think the convention for styled component props is to have a `$` prefix attached to them, so like `$color`, which is just to differentiate them from React components I think? Not too sure what conventions we're following though.

Reply:
Let's change to this! Thanks Jinkang